pkgs/top-level/all-packages.nix

```diff
@@ -18568,6 +18568,8 @@ in
 
   natron = callPackage ../applications/video/natron { };
 
+  NeoComp = callPackage ../applications/window-managers/NeoComp { };
```

Comment:
please make the attribute lowercase (`neocomp`), per: https://nixos.org/nixpkgs/manual/#sec-package-naming .

Actually I just pushed a commit fixing this but more importantly helping neocomp find its assets so it doesn't segfault on launch! :)

Reply:
That made it segfault for you?  Ha!

I didn't do this because it checks for `XDG_CONFIG_DIRS`, which seemed like a better way to give it these paths rather than patching.

I'm still a bit confused about §13.2.  Some parts of it seem to be in conflict, e.g. ‘try to stick to the upstream package name’ and ‘don't use uppercase letters in the `name` attribute’.  I've restructured the name/version after re-reading that section, though.

Reply:
Also re-squashed the commits to adhere to `CONTRIBUTING.md`.

Reply:
And fixed the OfBorg build.